frontend/prebuild/src/app/login/login.component.ts

```diff
@@ -17,13 +17,25 @@ export class LoginComponent implements OnInit {
 
   createRound() {
     $.post(`http://${document.location.hostname}:8080/round/create`, function(data) {
-      alert('Response is: ' + data);
+      alert('Created round: ' + data);
```

Comment:
Think we might wanna standardize on templating style--could avoid unexpected coercing of types.

```alert(`Created round: ${data}`);```